### DISCUSSION TITLE
Reuse default border style

### DISCUSSION THREAD

**Discussion by stivaugoin** [mantinedev/mantine#3228] [Q&A] [answered]
I want to create a custom component with border. The component must look like `Accordion` without the accordion 🙃

Is there a simple way to use border style (color, size) used anywhere else? I didn't find any border prop in MantineTheme

Thanks

**Comment by rtivital** [comment, marked as answer]
You can save any values to theme.other – https://mantine.dev/theming/theme-object/#other

**Reply by stivaugoin** [reply]
Thanks, it's not exactly what I need, but I didn't know about other. It will definitely help in my case.

🙏
